app/templates/suppliers/create_your_account_complete.html

```diff
@@ -32,13 +32,8 @@
       <h1 class="govuk-heading-l">Activate your account</h1>
 
       <div class="dmspeak">
```

Comment:
Can we get rid of this while we're here or does it look out of place with other pages on the journey?

Reply:
Oh I missed that, I will get rid of it